### DISCUSSION TITLE
Promotion to maintainer?

### DISCUSSION THREAD

**Discussion by miguelsousa** [fonttools/fontbakery#3869] [Q&A] [answered]
@felipesanches What's the process for getting promoted to repo maintainer? I'd like to be able to merge changes to the Adobe Fonts profile on my own. Thanks!

**Comment by felipesanches** [comment, marked as answer]
I've just sent you an invitation to join this repo as an "outside colaborator" with the "maintainer" role, the same access level shared by other collaborators like @cjchapman 

Your contributions to the FontBakery project are highly appreciated!
Thanks, @miguelsousa

**Reply by miguelsousa** [reply]
Perfect. Thanks!
